drivers/SmartThings/zigbee-thermostat/fingerprints.yml

```diff
@@ -13,12 +13,17 @@ zigbeeManufacturer:
     deviceLabel: Danfoss Thermostat
     manufacturer: Danfoss
     model: eTRV0100
-    deviceProfileName: thermostat-popp-danfoss
-  - id: "D5X84YU/eT093WRO"
```

Comment:
please leave the ids of existing fingerprints as-is

Reply:
👀

Reply:
```
  - id: "D5X84YU/eT093WRO"
    deviceLabel: POPP Smart Thermostat (Zigbee)
    manufacturer: D5X84YU
    model: eT093WRO
    deviceProfileName: thermostat-popp
  - id: "D5X84YU/eT093WRG"
    deviceLabel: POPP Smart Thermostat (Zigbee)
    manufacturer: D5X84YU
    model: eT093WRG
```
@greens so in the end it should look like this?
Sorry for the confusion our intention was to decide between Popp and Danfoss. In addition there were several Edge Driver beginner mistakes which we are trying to avoid by now. Thanks for your patience with us 👍

Reply:
your new fingerprint can have whatever id you want to use for it, I'd just prefer we not alter existing ones

Reply:
fixed with https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/285/commits/15e027cb7809ee16b4a3bf0ddbe35e7db448a077